decidim-core/lib/decidim/core/test/shared_examples/comments_examples.rb

```diff
@@ -56,10 +56,7 @@
       expect(page).to have_selector("#comment_#{deleted_comment.id}")
 
       expect(page).not_to have_content(deleted_comment.author.name)
-      expect(page).not_to have_content(deleted_comment.body.values.first)
-      # REDESIGN PENDING:
-      # When redesign is enabled in tests, the following line can be uncommented. The deleted text is not shown in the redesign.
-      # expect(page).to have_no_content(comment_body)
```

Comment:
The commented example has been uncommented and updated:
```diff
-      # expect(page).to have_no_content(comment_body)
+      expect(page).not_to have_content(translated(deleted_comment.body))
```
Because after some changes in this file the commented line became obsolete and comment_body left to be available

Reply:
Please let me resolve conversations to keep track of the things that are left open, specially if there isn't any change in the code and you're not actually making any changes. 


What do you mean with "comment_body left to be available"?

Seeing the history, this was introduced on #11122:

 https://github.com/decidim/decidim/blame/d15dcd6e7b073aaa53d985636969f27bd0b606c3/decidim-core/lib/decidim/core/test/shared_examples/comments_examples.rb#L62 

Seeing that this variable (`comment_body`) isn't defined in any of the context that this excample depends on, it seems like the introduction of these comments was a mistake, and we can ignore them. Is that it?

Reply:
Resolving this conversation so we can merge this PR, but please let me know if my last comment is alright of there's something else missing. 
This is an important issue as we're talking about the contents of a deleted comment and I want to be sure I understand why the original comment in the code was there. Thanks

Reply:
Looking at the history of this file in the #11122 branch it seems that both the comment and the commented example were added and the comment_body had no sense in that context, so you are right, the commented example never really made sense using `comment_body`. My mistake, I thought that the commented line came from the code before the redesign and had to be recovered because the purpose of the example made sense. I think that finally what I did was recover the example and seeing that it did basically the same thing as the previous line, I left only one (but I'm not totally sure if that was my mental process). Sorry for the misunderstanding

Reply:
No problem! I didn't want to let it pass just to be safe. Thanks for answering.